docker/docker-entrypoint.sh

```diff
@@ -8,6 +8,7 @@ $server -detached
 
 echo "Waiting for RabbitMQ to finish startup..."
 
+sleep 10
```

Comment:
It makes no sense to sleep before `rabbitmqctl await_startup`. That command waits for node boot to complete. User creation is synchronous, so I don't see any need to wait after we set up the permissions but at least it would make some sense there.

Reply:
Ok, as an unrelated change, I'll remove this again. No problem.

Reply:
Yes, we can address this separately if needed. Admittedly I do not use this image for testing as I usually test Bunny against RabbitMQ nodes built from source or GA releases running locally.